### DISCUSSION TITLE
[Project] presentation for 2 people team?

### DISCUSSION THREAD

**Discussion by haneul-yoo** [seominjoon/kaist-ai605#64] [Q&A] [answered]
Hi:)

I'm planning to participate in the project with one other student, then what should I do before the presentation?
Is there any team declaration file to submit?

If we're are doing a project in a team, should we each of us present individually? Or just do presentation once?

Regards,

**Comment by seominjoon** [comment, marked as answer]
You can do it at one of them and let the TA (Miyoung) know.  Thanks!
